### PR TITLE
Public Feed: added feed.html and feed-loader.js

### PR DESCRIPTION
**Pull request by ellikap6**
I added the feed.html page as given in the instructions. I want to do the suggested step and remove the javascript from the html page. The functions are very similar to the user page, so I would like to try encapsulation. The only difference is in the fetchMessages() function where the const url differs. I wanted to ask how you could make the function usable for both html pages.

### CODE REVIEW

File: src/main/webapp/js/feed-loader.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      return response.json();
+    }).then((messages) => {
+      const messageContainer = document.getElementById('message-container');
+      if(messages.length == 0){
+       messageContainer.innerHTML = '<p>There are no posts yet.</p>';
+      }
+      else{
+       messageContainer.innerHTML = '';  
+      }
+      messages.forEach((message) => {
```
Comment (tsalako) [inline src/main/webapp/js/feed-loader.js]: Can we move this into the else statement? It wont run for message length 0 as is, but its a bit easier to read inside the else.

**Review by tsalako — CHANGES_REQUESTED** [review]
This looks great. Can you attach a screenshot of this feature so we can see what it looks like.

**Comment by ellikap6** [comment]
screenshot 23

Here's the screenshot :)

File: src/main/webapp/js/feed-loader.js (hunk 1 above)
Comment (ellikap6) [reply src/main/webapp/js/feed-loader.js]: I made this change

**Review by tsalako — APPROVED** [review]

**Pull request merged**